frontend/src/components/GameboardCell.tsx

```diff
@@ -39,6 +39,7 @@ const GameboardCell: React.FunctionComponent<Props> = ({
 
   return (
     <td
+      id={unit && `board-unit-${unit.id}`}
```

Comment:
Plase change this to `const unitId = useState(unit.id)`. I found out this kind of interpolation can be troublesome with redux state management, and slower.

Reply:
Ok, sure

Reply:
No, it doesn't work... Use state only runs on the initial render, when the props change it doesn't run useState again. My implementation works fine here

Reply:
Ahh, ok, then my bad, sorry. :)